mobile/cmp/header/AppMenuModel.js

```diff
@@ -64,6 +66,12 @@ export class AppMenuModel extends MenuModel {
                 action: () => XH.showImpersonationBar(),
                 prepareFn: (item) => item.hidden = !XH.identityService.canImpersonate
             },
+            {
+                icon: Icon.info(),
+                text: `About ${XH.appName}`,
```

Comment:
Minor point - should we use `XH.clientAppName` here?  I think that might be preferable in some cases where the `appName` is the overall project and might not be familiar to end-users.